Index: Player.java
===================================================================
diff --git a/Player.java b/Player.java
deleted file mode 100644
--- a/Player.java	(revision b682fe51c5673ee72ff55ab105b03ff507c5fdad)
+++ /dev/null	(revision b682fe51c5673ee72ff55ab105b03ff507c5fdad)
@@ -1,25 +0,0 @@
-public class Player {
-    King King = new King();
-    Queen Queen = new Queen();
-    Pawn PawnOne = new Pawn();
-    Pawn PawnTwo = new Pawn();
-    Bishop BishopOne = new Bishop();
-    Bishop BishopTwo = new Bishop();
-    Knight KnightOne = new Knight();
-    Knight KnightTwo = new Knight();
-    Rook RookOne = new Rook();
-    Rook RookTwo = new Rook();
-    Rook RookThree = new Rook();
-    Rook RookFour = new Rook();
-    Rook RookFive = new Rook();
-    Rook RookSix = new Rook();
-    Rook RookSeven = new Rook();
-    Rook RookEight = new Rook();
-
-
-    String playerName;
-
-    public Player(String name) {
-        this.playerName = name;
-    }
-}
